packages/ui-theme-tokens/src/instructure/colors.ts

```diff
@@ -23,9 +23,9 @@
  */
 
 import { functionalColors } from '../utils/functionalColors'
-import { BaseColors, Colors } from '@instructure/shared-types'
+import { Colors } from '@instructure/shared-types'
 
-const values: BaseColors = {
+const values: any = {
```

Comment:
no `any` please

Reply:
I just left it in to make ts pass, I knew we will delete the whole instructure theme folder anyways